Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3233,6 +3233,15 @@ void LocalFrameView::updateTiledBackingAdaptiveSizing()
     tiledBacking->setScrollability(computeScrollability());
 }
 
+void LocalFrameView::allowHDR(bool allow)
+{
+    auto* tiledBacking = this->tiledBacking();
+    if (!tiledBacking)
+        return;
+
+    tiledBacking->allowHDR(allow);
```

Comment:
This only does the main tiled layer. What about all the other compositing layers, and video?